### ISSUE TITLE
Nearest elapse for daily OnCalendar= timers appears to be skipped after timer activation

### ISSUE THREAD

**Issue by intelfx** [systemd/systemd#3321]
### Submission type
- [X] Bug report
- [ ] Request for enhancement (RFE)
### systemd version the issue has been seen with

229.r1113.g508c45d
### Used distribution

Arch
### Steps to reproduce the problem

```
$ systemctl --user cat alarm-morning-early.timer       
# /home/intelfx/.config/systemd/user/alarm-morning-early.timer
[Unit]
Description=Morning alarm

[Timer]
OnCalendar=Mon-Fri 08:00:00
WakeSystem=true
Unit=alarm.service

[Install]
WantedBy=timers.target
```

```
$ LC_ALL=C date                                 
Sun May 22 03:51:31 MSK 2016
```

```
$ systemctl --user restart alarm-morning-early.timer; LC_ALL=C systemctl  --user list-timers alarm-morning-early.timer
NEXT                         LEFT          LAST PASSED UNIT                      ACTIVATES
Mon 2016-05-23 08:00:00 MSK  1 day 4h left n/a  n/a    alarm-morning-early.timer alarm.service
```
### In case of bug report: Expected behaviour you didn't see

"4h left"
### In case of bug report: Unexpected behaviour you saw

"1 day 4h left"

**Comment by intelfx** [comment]
Bisected to 436dd70f5331ec541ebdd84e106abaa63203f6f1 "calendarspec: sub-second support, v3".

**Comment by intelfx** [comment]
Oh, wow, looks like this was fixed in a022d76e6a7c711954be5535e6308d0a470e232a...
